server/src/actors/api/inbound_message/mod.rs

```diff
@@ -0,0 +1,7 @@
+mod connect;
+mod global_chat;
+mod reconnect;
+
+pub use connect::Connect;
```

Comment:
Since this will be an application rather than library, I think `pub(crate)` would be fine here.

Reply:
I've used pub, since the `actors/mod.rs` doesn't expose them publicly but just locally, you cannot access it outside actors module. 

(Spent some time figuring out how the modules and visibility works. You can make it as public as you want, but if parent module doesn't expose it publicly as well, it is invisible outside.)

Reply:
Ah, I see. I didn't realize it's that deep, np then :+1: